clients/cloud/nodejs/README.md

```diff
@@ -0,0 +1,74 @@
+# Overview
+
+Produce messages to and consume messages from [Confluent Cloud](https://www.confluent.io/confluent-cloud/) using the [ node-rdkafka client for Apache Kafka](https://github.com/Blizzard/node-rdkafka).
+
+# Prerequisites
+
+* [Node.js](https://nodejs.org/) version 8.6 or higher installed on your machine.
+* [OpenSSL](https://www.openssl.org) version 1.0.2.
+* Access to a [Confluent Cloud](https://www.confluent.io/confluent-cloud/) cluster
+* Local file with configuration parameters to connect to your Confluent Cloud instance ([how do I find those?](https://docs.confluent.io/current/cloud/using/config-client.html#librdkafka-based-c-clients)). Format the file as follows:
+```bash
+$ cat ~/.ccloud/example.config
+bootstrap.servers=<broker-1,broker-2,broker-3>
+sasl.username=<api-key-id>
+sasl.password=<secret-access-key>
+```
+* Users of macOS 10.13 (High Sierra) and above may need to specify OpenSSL paths so that the client's [librdkafka](https://github.com/edenhill/librdkafka) bindings can be built correctly:
```

Comment:
@colinhicks is this requirement unique to Node.js or all clients?

Reply:
Actually, perhaps all of this can be cut from this README, and instead provide a pointer to https://github.com/Blizzard/node-rdkafka#mac-os-high-sierra--mojave ?

Reply:
Sure, we could link the client documentation instead, @ybyzek.

I think it's okay to consider this setup step scoped to node-rdkafka specifically, in that it's the mechanism that library uses to override the default linkage, here: https://github.com/Blizzard/node-rdkafka/blob/64fdf8daccd5492ae39a3e6119b8404c52232000/binding.gyp#L134-L144. 

My sense is that one of the following applies to this issue, across librdkafka libs:
* The library requires a similar setup step, and we should expect it to be part of that library's documentation. (this case)
* The library handles this issue transparently. (e.g. https://github.com/appsignal/rdkafka-ruby/blob/4c320b4df963a6118f45f3910042cffd685cdd1d/ext/Rakefile#L26-L29)
* The library doesn't yet handle it, but given any scale of user base, the issue will be raised.

Reply:
@ybyzek Sorry, I just realized that you suggested a pointer to the more general `#requirements` anchor. I made the previous comment and README change w/r/t the OpenSSL issue. And I missed your point that a bigger chunk of the reqs could be delegated to node-rdkafka, although we do require a more recent Node.js version in our example.

Let's just do whatever you think strikes the best balance :).

Reply:
@colinhicks generally speaking I prefer to de-dupe.  If there is another page that has the full set of install requirements/steps, then I prefer to link to that instead of repeating it.

So do you think https://github.com/Blizzard/node-rdkafka#requirements or https://github.com/Blizzard/node-rdkafka/blob/56c31c4e81f2a042666160338ad65dc4f8f2d87e/README.md#mac-os-high-sierra--mojave is complete?  If yes, I prefer to just link to one of those pages, and from this README we'd remove the `npm install`, etc commands.

Reply:
Ah, my answer to that question is a qualified no. I think the current readme is actually reasonable and not unlike other node examples in the wild.

In particular the `npm install` portion is important to keep in situ. It's similar in pattern to the Ruby example's Bundler instructions.

The original rub that led us here is the macOS/OpenSSL binding issue. It's an unfortunate wart. But I think it's fine to cover it with the link as-is.

Reply:
@colinhicks makes sense.  In that case, for your consideration, move the two bullet points about `npm install` and `macOS` to be higher up in the bullet list because they are more closely related to the Node.js install (could even be indented bullets?).  Last item in the list should be about the ccloud config.